src/components/Dialog.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { createPortal } from 'react-dom';
+import FocusTrap from 'focus-trap-react';
+import MovieForm from './MovieForm';
+import { InitialMovieInfoInterface } from '../App';
+
+interface DialogProps {
+  title: string;
+  initialMovieInfo: InitialMovieInfoInterface;
+  onSubmitDialog: (movieInfo: InitialMovieInfoInterface) => void;
+  onCloseDialog: () => void;
+}
+
+const Dialog: React.FC<DialogProps> = ({ title, initialMovieInfo, onSubmitDialog, onCloseDialog }) => {
```

Comment:
according to task requirements Dialog should accept JSX for body content in "children" property. Use `React.FC<PropsWithChildren<DialogProps>>`